### DISCUSSION TITLE
How to recreate the location information of all accessed IPs

### DISCUSSION THREAD

**Discussion by missuo** [shlinkio/discussions#102] [Help wanted] [answered]
#### How Shlink is set up

* Shlink Version: 3.5.1
* PHP Version: 8.1
* How do you serve Shlink: Self-hosted nginx
* Database engine used: MySQL

#### Summary

Hello, I didn't enter Maxmind key in previous 3.4 version, but I configured the key before updating to 3.5.1, but after I updated it, I found that the location information is only available for the access logs after the update. I would like to know how to regenerate the location information for all visits (including the visits before the update)?

**Comment by acelaya** [comment, marked as answer]
Try running `bin/cli visit:locate --retry`.

It will try to take all existing visits that have not been geolocated yet and do it.

The only consideration is that it will do it with the anonymized version of the IP address, so you'll lose a bit of accuracy, but it should be negligible and there's nothing to do about it anyway.

**Comment by missuo** [comment]
Thanks a lot. I executed it, and it displays properly. @acelaya
